images/access-setup/content/bin/setup_compute.sh

```diff
@@ -149,17 +183,22 @@ generate_compute_credentials() {
   get_context "pac-manager" "pipelines-as-code" "pac-manager" "$kubeconfig"
   printf "%s\n" "$kubeconfig"
 
-  mkdir -p "$WORK_DIR/environment/compute/$compute_name"
-  echo "resources:
-  - $GIT_URL/gitops/argocd?ref=$GIT_REF
-" >"$WORK_DIR/environment/compute/$compute_name/kustomization.yaml"
+  printf "    - Generate kustomization.yaml: "
+  manifests_dir="$WORK_DIR/environment/compute/$compute_name"
+  mkdir -p "$manifests_dir"
+  echo -n "---
+resources:
+  - git::$GIT_URL/gitops/argocd?ref=$GIT_REF
```

Comment:
could you please help me understand what git:: does?

Reply:
It's a hint to kustomize understand that this is a repository. While it seems to be optional most of the time, it didn't pick up the url as a repo, thus the need for the hint.